src/service/service.ts

```diff
@@ -283,6 +288,40 @@ export class Discv5 extends (EventEmitter as { new (): Discv5EventEmitter }) {
     });
   }
 
+  /**
+   * Broadcast TALKREQ message to all nodes in routing table
+   */
+  public async broadcastTalkReq(payload: Buffer, protocol: string | Uint8Array): Promise<void> {
```

Comment:
Buffer -> Uint8Array

Reply:
Sorry brain fart on that one.  I've corrected both this and the one above and verified that everything is still working with my client app so far.